### PR TITLE
md modifications

### PR DESCRIPTION
**Pull request by salmanmkc**
## Fixes
Small md fixes, GitHub capitalisation and wording

## Type of PR
Corrective 

## What's happening now?
No change

## What's changed?
Just md files

## Checklist
[] Tested in deployment environment
[] Tested in an alternate environment (please update here: )
[] Major feature added
[] Bux fix
[x] Other (explain: Corrective)

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ This information will help us triage your report more quickly.
 
 ## Preferred Languages
 
-We prefer all communications to be in English.
+We prefer all communications to be done in English.
```
Comment (pdeligia) [inline SECURITY.md]: Could you please revert this change? This wording was intended.

Comment (salmanmkc) [reply SECURITY.md]: Sure

File: SECURITY.md [hunk 2]
```diff
@@ -32,4 +32,4 @@ We prefer all communications to be in English.
 
 Microsoft follows the principle of [Coordinated Vulnerability Disclosure](https://www.microsoft.com/en-us/msrc/cvd).
 
-<!-- END MICROSOFT SECURITY.MD BLOCK -->
\ No newline at end of file
+<!-- END MICROSOFT SECURITY.MD BLOCK -->
```
Comment (pdeligia) [inline SECURITY.md]: Could you please revert this file change?

Comment (salmanmkc) [reply SECURITY.md]: I don't know why that is coming up as a change, there's no rich difference

Comment (pdeligia) [reply SECURITY.md]: It could be a weird line ending change, but we should revert it before we can merge the PR! You could perhaps rebase and drop this file completely from the PR to fix this.

File: Versioning.md [hunk 3]
```diff
@@ -1,32 +1,37 @@
 # Guidance on Coyote versioning
```
Comment (pdeligia) [inline Versioning.md:1]: Thanks for reverting the security file! And also for the "*" to "-" and "1)" to "1." (although they don't actually make any difference in the markdown output).

File: Versioning.md [hunk 4]
```diff
@@ -1,32 +1,37 @@
...
- * anything else changes in the the Coyote framework.
+
+- anything else changes in the the Coyote framework.
 
 Not all changes to the repository warrant a version change. For example,
- * test code changes
- * documentation only fixing typos and grammar
- * automation script updates
- * reformatting of code for styling changes
+
```
Comment (pdeligia) [inline Versioning.md:30]: Could you revert this line change so there is no new line added?

File: Versioning.md [hunk 5]
```diff
@@ -1,32 +1,37 @@
...
 **Incrementing the MINOR version number** should be done when:
- * new features are added to the Coyote API or tools that are backwards
- compatible
+
+- new features are added to the Coyote API or tools that are backwards
+  compatible
 
 **Incrementing the PATCH version number** should be done when
- * anything else changes in the the Coyote framework.
+
```
Comment (pdeligia) [inline Versioning.md:26]: Same here, please revert this to not have a new line added. Makes no difference to the output.

File: Versioning.md [hunk 6]
```diff
@@ -1,32 +1,37 @@
...
+3. PATCH version when you make backwards compatible bug fixes.
 
 We adopt everything listed in [Semantic Versioning 2.0.0](https://semver.org/)
 but to summarize the major points here:
 
 **Incrementing the MAJOR version number** should be done when:
- * a major new feature is added to the Coyote framework or tools and is showcased by a new tutorial demonstrating this feature
- * a breaking change has been made to Coyote API or serialization format, or
- tool command line arguments.
+
```
Comment (pdeligia) [inline Versioning.md:15]: Same here, please revert this to not have a new line added.

File: Versioning.md (hunk 3 above)
Comment (pdeligia) [reply Versioning.md:1]: I will approve and merge the PR once you revert the changes related to the few comments that I added.

File: SECURITY.md (hunk 2 above)
Comment (pdeligia) [reply SECURITY.md]: Thanks!

File: Versioning.md (hunk 3 above)
Comment (salmanmkc) [reply Versioning.md]: Should all be resolved now

Comment (pdeligia) [reply Versioning.md]: Thanks!
